spec/classes/collectd_plugin_disk_spec.rb

```diff
@@ -48,6 +48,25 @@
     end
   end
 
+  context ':manage_package => false on osfamily => RedHat' do
+    let :facts do
+      {
+        osfamily: 'RedHat',
+        collectd_version: '5.5',
+      }
+    end
+
+    let :params do
+      {
+        manage_package: false,
+      }
+    end
+    it 'Will not manage collectd-disk' do
+      should_not contain_package('collectd-disk').with(ensure: 'present',
```

Comment:
You can probably leave off the `.with` bit here as we don't care what the parameters are; just that the catalog doesn't contain any package resource for 'collectd-disk'